styles.css

```diff
@@ -1,15 +1,19 @@
 body {
-  background-image: url(/Image/background.png);
-  background-repeat: no-repeat;
-  background-size: 375px 590px;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
 }
```

Comment:
Great job so far @Solobarine 👏

- [ ] Just like the previous reviewer pointed out, you may please fix this white space.
- Your design
![white space](https://user-images.githubusercontent.com/63917725/177010739-48786448-e86b-455d-9dc0-1d8e3153585f.PNG)